application-services/custom/influxdb-export/res/configuration.yaml

```diff
@@ -0,0 +1,38 @@
+Writable:
+  LogLevel: "INFO"
+  InsecureSecrets:
+    mqtt:
+      path: "mqtt"
+      Secrets:
+        username: "edgex"
+        password: "password"
+        cacert: ""
+        clientcert: ""
+        clientkey: ""
+  Writable.Telemetry:
+    Interval: "30s"
+    PublishTopicPrefix : "edgex/telemetry" # /<service-name>/<metric-name> will be added to this Publish Topic prefix
+
+Service:
+  Host: "localhost"
+  Port: 59780 # Adjust if running multiple examples at the same time to avoid duplicate port conflicts
+  StartupMsg: "App service export to InfluxDB has Started"
+
```

Comment:
Missing MessageBus section for client ID override

```
MessageBus:
  Disabled: false  # Set to true if not using edgex-messagebus Trigger below and Metrics are enabled
  Optional:
    ClientId: "app-influxdb-export"
```

Reply:
done